README.md

```diff
@@ -31,6 +31,7 @@ ownCloud is an open-source file sync, share and content collaboration software t
 
 ## Docker Tags and respective Dockerfile links
 
+- [`10.12.0-rc2`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:10.12.0-rc2`
```

Comment:
As always, please don't add RC releases to the README.

Reply:
ok. the instruction says 'all' to te README. I'll change that.